deployment/SampleClaimsProvider/Controllers/V20210519/ClaimsProviderController.cs

```diff
@@ -0,0 +1,136 @@
+﻿// ---------------------------------------------------------------------------------
+// <copyright company="Microsoft">
+//   Copyright (c) {Microsoft} Corporation. All rights reserved.
+// </copyright>
+// ---------------------------------------------------------------------------------
+
+namespace SampleClaimsProvider
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Collections.ObjectModel;
+    using System.IO;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.Logging;
+    using Microsoft.Extensions.Primitives;
+    using Newtonsoft.Json;
+
+    [Route("api/claims/claimInfo")]
+    [ApiController]
+    public class ClaimsProviderController : ControllerBase
+    {
+        private readonly IClaimsProviderStore claimsProviderStore;
+        private readonly ILogger<ClaimsProviderController> logger;
+
+        public ClaimsProviderController(ILogger<ClaimsProviderController> logger, IConfiguration configuration, IClaimsProviderStore claimsProviderStore)
+        {
+            this.logger = logger;
+            this.claimsProviderStore = claimsProviderStore;
+        }
+
+        /// <summary> This API retrieves claims for a specified vehicleId.
+        /// <example>
+        /// [GET] https://{hostname}/api/claims/claimInfo/vehicles/testVehicle1?paths=path1
+        /// </example>
+        /// </summary>
+        [HttpGet]
+        [Route("vehicles/{vehicleId}")]
+        public async Task<IActionResult> RetrieveClaimsWithVehicleId(string vehicleId)
+        {
+            return await this.RetrieveClaimsAsync(this.Request, vehicleId, userId: null);
+        }
+
+        /// <summary> This API retrieves claims for a specified vehicleId and userId.
+        /// <example>
+        /// [GET] https://{hostname}/api/claims/claimInfo/vehicles/testVehicle1/users/testUser1?paths=path1
```

Comment:
Same thing about documenting a use case to get all of the claims. I believe its `paths=*`